libs/ui-watt/src/lib/components/button/watt-button.stories.ts

```diff
@@ -73,7 +73,9 @@ Button.args = {
 const iconButtonTemplate = (args: Partial<WattIconButtonComponent>) =>
   `<watt-icon-button icon="${args.icon}" [disabled]="${args.disabled}" label="Something meaningful"></watt-icon-button>`;
 
-const IconButton: Story<WattIconButtonComponent> = (args) => ({
+const IconButton: Story<WattIconButtonComponent | WattButtonComponent> = (
```

Comment:
question: Was there a problem with `WattButtonComponent` not being specified?

Reply:
Compilation error, if not specified we can't disable the desired controls because they are not part of WattIconButtonComponent